pkgs/applications/video/hypnotix/default.nix

```
@@ -0,0 +1,88 @@
+{ stdenv, lib, fetchFromGitHub, dconf, gettext, glib
+, gsettings-desktop-schemas, gtk3, hicolor-icon-theme
+, mpv, python3, python3Packages, wrapGAppsHook
```

```suggestion
, mpv, python3, wrapGAppsHook
```
maker sure we do not mix python versions.